helm/prometheus-rules/templates/alerting-rules/draughtsman.rules.yml

```diff
@@ -20,7 +20,7 @@ spec:
       labels:
         area: empowerment
         severity: notify
-        team: biscuit
+        team: honeybadger
```

Comment:
IMO this alert should just be removed?

Reply:
up to teams to kill any dumb alerts, this PR is already too big :D